source/manual/debian-packaging.html.md

```diff
@@ -10,6 +10,94 @@ review_in: 6 months
+
+Due to limited availability of up-to-date packages in long term support (LTS)
+distributions, requirements of software may make installation of a package
+outside of the distribution infrastructure (currently Ubuntu "Trusty" 14.04
+LTS) necessary.
+
+GDS developers may also choose to use the Debian packaging system as a means to
+distribute and maintain their locally developed software.
+
+As always when considering the addition of new software, the first question has to be:
+`is this really necessary?` Adding a package should be motivated by
+security or architecture requirements and aim to minimise unnecessary
+dependencies.
+
+Additional guidance can be found in the [GDS way](https://gds-way.cloudapps.digital/standards/tracking-dependencies.html#dependency-management-tools)
+as well as the [service manual](https://www.gov.uk/service-manual/technology/managing-software-dependencies#how-to-work-with-third-party-code).
+
+If you are satisfied that adding a local or third party package vi aptly
+optimal solution, there are a number of ways packages may be sourced and added,
+all carrying different advantages and disadvantages as well as security and
+maintenance implications.
+
+### Creating and maintaining packages locally.
+
+- Creating local packages allows for the greatest flexibility with respect to
+  installation, default configuration and fullfilment of requirements.
+
+- Locally developed bug-fixes can be included and distributed independent of
+  distribution services.
+
+- Distribution bug-fixes and updates are not available and the package has to
+  be actively maintained by GOV.UK.
+
+- To minimise technical debt on future maintenance of packages, locally created
+  packages should only be considered if there is no reliable and secure third
+  party option available.
+
+### Mirroring packages maintained by a third party.
+
+- If considering the use of a package created by a third party, the
+  trustworthiness of said party is critical.
+
+- General Debian packages explicitly labelled as suitable for Ubuntu Trusty,
+  such as Ubuntu packages provided by PostgreSQL or PPA packages by LibreOffice
+  can be considered safe.
+
+- When considering private contributors and/or smaller companies contributions
+  both the maturity of the provided software and trust for the source need to be
+  considered. Comments on maturity as well as the number of users of a package
+  may be an indicator of its suitability for use in production.
+
+- Development activity and update frequency are additional factors to consider.
+
+- When in doubt about the trustworthiness or stability of a third party package,
+  this option should be disregarded in favour of other solutions not
+  compromising on reliability and security.
+
+### Mirroring or importing Debian packages created for other distributions.
+
+- If great care is taken, it may be possible to make use of a software version
+  which has been packaged for another Linux distribution, e.g. Debian.
```

Comment:
Ubuntu is based on Debian, and these docs introduce "Debian packaging". Was this the "other distribution" you wanted to give?